g2p/mappings/langs/eng/README.md

```diff
@@ -0,0 +1,10 @@
+Aligned CMUDict for G2P
+-----------------------
+
+The alignments in `cmudict_SPHINX_40.ipa.aligned.txt` were generated by
+`rustfst-g2p`:
+
+    python make_ipa_cmudict.py > tmp.txt
```

Comment:
I like this tmp file, I generated it on my computer. So much easier to read the English dict in IPA than in ARPABET!